src/server/controllers/account-transactions.js

```diff
@@ -181,6 +184,18 @@ const formatAmountAsString = (amount) => {
   return `${amountAsString} ${amount.currency}`;
 };
 
+const formatAccountName = (account) => {
+  const legalName = account?.legalName;
+  const name = account?.name;
+  if (!legalName && !name) {
+    return '';
+  } else if (legalName && name) {
+    return `${legalName} (${name})`;
+  } else {
+    return legalName || name;
+  }
+};
+
```

Comment:
Can we maybe handle the case where legalName === name and skip name in this case?

Already 86 entries there:

```
SELECT * FROM "Collectives" WHERE "legalName" = "name";
```

Reply:
Good catch, I changed that. I'll bring this change in frontend and PDF service as well